UPGRADE.md

```diff
@@ -0,0 +1,105 @@
+# Upgrade
+
+## 2.1
+
+### Switch from LiipThemeBundle to SyliusThemeBundle
+
+To achieve Symfony 5 compatibility the ThemingBundle has to be changed from [LiipThemeBundle](https://github.com/liip/LiipThemeBundle) to [SyliusThemeBundle](https://github.com/Sylius/SyliusThemeBundle).
+
+#### Remove the old theme bundle and install the SyliusThemeBundle:
+
+```bash
+# Remove old theme-bundle
+composer remove liip/theme-bundle --no-update
+
+# Install new theme-bundle
+composer require sylius/theme-bundle
+```
```

Comment:
Add a docs that you need to convert the `liip_theme:` config into a `theme.json` configuration. Would work here with a specific example so show the old liip_theme config:

```diff
-liip_theme:
-    themes: ['project', 'awesome']
-    active_theme: 'awesome'
```

Which should then be removed and converted into a `theme.json`:

```json
{
    "name": "app/awesome"
}
```

Showing the structure and at the end the sulu specific configuration in the webspace:

```diff
-<theme>awesome</theme>
+<theme>app/awesome</theme>
```